irods/test/data_obj_test.py

```diff
@@ -2179,7 +2179,7 @@ def test_temporary_xml_mode_change_with_operation_as_proof__issue_586(self):
 
         sess = irods.test.helpers.make_session()
         hc = home_collection(sess)
-        odd_name = "{hc}/\1".format(**locals())
+        odd_name = r"{hc}/\1".format(**locals())
```

Comment:
For this test, it's important  that we keep the \1 as a non-raw character, so let's drop this part of the diff.

Reply:
Oops, I missed that `\1` is a valid octal escape sequence for `\x01`. Thanks for catching that!

Reply:
I’m about to force-push the correction. I just skimmed the diff and didn’t see any other similar errors.